components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/utils/SCIMCommonConstants.java

```diff
@@ -100,5 +100,10 @@
     public static final String ENABLE_LOGIN_IDENTIFIERS = "LoginIdentifiers.Enable";
     public static final String PRIMARY_LOGIN_IDENTIFIER_CLAIM = "LoginIdentifiers.PrimaryLoginIdentifier";
     public static final boolean DEFAULT_ENABLE_LOGIN_IDENTIFIERS = false;
+
+    // Date of Birth related constants.
+    public static final String DATE_OF_BIRTH_LOCAL_CLAIM = "http://wso2.org/claims/dob";
+    public static final String DATE_OF_BIRTH_REGEX = "^\\d{4}-\\d{2}-\\d{2}$";
```

Comment:
can't we get this regex from claim metadata?

Reply:
addressed in 7b3fb87